packages/core/src/domain/configuration/experimentalFeatures.ts

```diff
@@ -31,3 +31,7 @@ export function isExperimentalFeatureEnabled(featureName: string): boolean {
 export function resetExperimentalFeatures(): void {
   enabledExperimentalFeatures = new Set()
 }
+
+export function getExperimentalFeatures(): Set<string> {
+  return enabledExperimentalFeatures || []
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** Here, you might return an array instead of a `Set`.

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** To spot such issue, change the `enabledExperimentalFeatures` type to `Set<string> | undefined`